src/text.ts

```diff
@@ -1,8 +1,6 @@
-/* @flow strict */
-
 export function insertText(textarea: HTMLInputElement | HTMLTextAreaElement, text: string): void {
-  const beginning = textarea.value.substring(0, textarea.selectionStart)
-  const remaining = textarea.value.substring(textarea.selectionEnd)
+  const beginning = textarea.value.substring(0, textarea.selectionStart || 0)
+  const remaining = textarea.value.substring(textarea.selectionEnd || 0)
```

Comment:
(2/2) `selectionStart` and `selectionEned` only applies to input elements of certain types. https://html.spec.whatwg.org/#do-not-apply